front/src/common/api/osrdEditoastApi.ts

```diff
@@ -185,8 +185,7 @@ const osrdEditoastApi = generatedEditoastApi
           }
           return { data: simulation };
         },
-        // TODO: fix getPacedTrainByIdSimulation tags (it should be paced_train)
-        providesTags: ['train_schedule'],
+        providesTags: ['train_schedule', 'paced_train'],
```

Comment:
We should set these depending on the type of train ID in the request, instead of providing both tags. See e.g. `getTimetableItemById`.

I'd prefer to keep a TODO until this is done.